### ISSUE TITLE
Point to "Confluent for Kubernetes Scenario Examples" repo

### ISSUE THREAD

**Issue by chuck-confluent** [confluentinc/examples#1060]
I think a lot of folks come here to look for examples, and there is a kubernetes folder that shows a demo. That's great, but I think there should also be a link to https://github.com/confluentinc/confluent-kubernetes-examples which is a pretty comprehensive demo repo for CfK

**Comment by ybyzek** [comment]
@chuck-confluent agree, and even more so, perhaps https://github.com/confluentinc/examples/tree/7.0.1-post/kubernetes should be deprecated entirely because it uses older version of Confluent Operator.  Would be great to get PM input on this (@amitkgupta ?)

**Comment by amitkgupta** [comment]
@ybyzek I think the demos in this repo are nice because (a) they include the Kubernetes cluster setup part too -- we have folks ask us from time to time for exactly this, the full flow on AWS or Azure starting from nothing, whereas confluent-kubernetes-examples assumes you BYOKubernetes, and (b) they have the high fidelity docs.confluent.io treatment with a more Confluent-native look and feel. We should aim to update them to use the new CFK rather than deprecating them.

@chuck-confluent thinking this through, maybe rather than linking to that other repo, the content from that repo should be merged into this one. As long as it's okay that some demos have the user walk through the actual commands the product offers without the `make` wrappers and some have the `make` wrappers, and if it's okay that some have the high-fidelity docs and some are just GitHub Markdown files, I think the two types can co-exist here nicely.

@rohit2b @arodoni wdyt?
